whiskey.yml

```diff
@@ -35,6 +44,7 @@ Publish:
                 ReleaseNotes: RELEASE_NOTES
 
 - GitHubRelease:
+    UnlessExists: env:APPVEYOR_PULL_REQUEST_NUMBER

```

Comment:
Should the `GitHubRelease` task have an `OnlyOnBranch: master` property? Our pattern seems to be not to publish beta builds via GitHub releases.

Reply:
This was so that we weren't publishing a whole bunch of beta releases to GitHub. Now that we're more intentionally publishing betas to the `prerelease` branch, I don't see a problem with also publishing those betas to GitHub.

Reply:
Ah yes, that's right. I like it.